packages/frontend/src/components/account/JoinHouseModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button, Input, Modal, Text } from '@nextui-org/react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { HouseServices } from '../../services/HouseService';
+
+interface CreateHouseModalProps {
+  joinVisible: boolean;
+  setJoinVisible(value: boolean): void;
+}
+const JoinHouseModal: React.FC<CreateHouseModalProps> = ({
+  joinVisible,
+  setJoinVisible,
+}) => {
+  const closeJoinHandler = () => {
+    setJoinVisible(false);
+  };
+
+  const [houseData, setHouseData] = useState();
```

Comment:
I dont think this hook is needed (houseData is never used, only the setter, so neither is probably needed)